### PR TITLE
chore: add laravel/framework 9 to composer.json

### PR DESCRIPTION
**Pull request by mhabedini**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,15 +13,15 @@
...
+        "laravel/framework": "^7.0|^8.0|^9.0",
         "guzzlehttp/guzzle": "^7.0",
         "behamin/bresources": "^2.2|^1.5"
     },
     "scripts": {
         "test": "phpunit --color=always"
     },
     "require-dev": {
-        "orchestra/testbench": "^6.0"
+        "orchestra/testbench": "^v6.0|^v7.0"
```
Comment (omalizadeh) [inline composer.json:24]: add composer.lock to gitignore

File: src/Requests/PendingRequest.php [hunk 2]
```diff
@@ -74,7 +74,7 @@ public function patch($url, $data = [])
         return $this->respond($url, $data, Request::METHOD_PATCH);
     }
 
-    public function post(string $url, array $data = [])
+    public function post(string $url, $data = [])
```
Comment (omalizadeh) [inline src/Requests/PendingRequest.php:77]: why u did this?

**Review by omalizadeh — COMMENTED** [review]
orchestra testbench is being installed on local laravel projects with bfilter!

File: composer.json (hunk 1 above)
Comment (mhabedini) [reply composer.json:24]: It's already in .gitattributes as export-ignore

File: src/Requests/PendingRequest.php (hunk 2 above)
Comment (mhabedini) [reply src/Requests/PendingRequest.php:77]: Because this argument signature should be same as the parent class method.

File: composer.json (hunk 1 above)
Comment (omalizadeh) [reply composer.json:24]: gitignore not attributes. when you support testbench 6 and 7 so you don't care which version he is using. so why you should lock in composer.lock and push it on github?!

File: src/Requests/PendingRequest.php (hunk 2 above)
Comment (omalizadeh) [reply src/Requests/PendingRequest.php:77]: i think there is no problem if your arg is more specific in extended class. BUT it's not correct the other way.

**Pull request closed without merging**